src/bittytax/conv/parsers/nexo.py

```diff
@@ -77,8 +77,16 @@ def parse_nexo(data_row: "DataRow", parser: DataParser, **_kwargs: Unpack[Parser
         value = None
 
     if row_dict["Type"] in ("Deposit", "Top up Crypto", "ExchangeDepositedOn"):
+        # Skip credit deposits (already handled with "Loan Withdrawal").
+        if row_dict["Details"].find("Credit") > -1:
+            return
+        elif row_dict["Details"].find("Airdrop") > -1:
```

Comment:
Please can you provide an "Airdrop" example row?

Reply:
Yep.